public/usage-examples/geometry/point_at/point_at-1-simple.txt

```diff
@@ -0,0 +1,3 @@
+#### Draw face
+
+The following code shows examples of using [Point At](/api/geometry/#point-at) to show how to create a point at a specific location and draw different graphs base on the point.
```

Comment:
This could be better. If you are drawing faces maybe write: 
The following code shows an example of using [Point At](/api/geometry/#point-at) to show how to create a point at a random location and draw faces at that point.